src/Platform/Microsoft.Testing.Platform/Logging/FileLoggerProvider.cs

```diff
@@ -16,6 +18,14 @@ internal sealed class FileLoggerProvider : ILoggerProvider, IDisposable
     private readonly string _logPrefixName;
     private readonly bool _customDirectory;
 
+    /* for testing */
+    internal FileLoggerProvider()
```

Comment:
Nit: suggested syntax avoid having the comment being moved around and ending up not attached.

```suggestion    
    internal /* for testing */ FileLoggerProvider()
```